storybook/storybook-react/src/PageHeading/PageHeading.docs.mdx

```diff
@@ -6,12 +6,14 @@ import README from "../../../../packages/css/src/components/page-heading/README.
 
 <Markdown>{README}</Markdown>
 
+## Examples
```

Comment:
We generally use this heading after the Primary and Controls blocks, no?

Reply:
No, wasn’t my idea. It’s the primary example, right?

Reply:
Hm, I see we don't really follow a convention here. Sometimes ## Examples is above the Primary story, sometimes it's below. And sometimes we don't use it at all 😅 Let leave it for now and discuss this in [this ticket](https://gemeente-amsterdam.atlassian.net/browse/DES-598)